Install/sql/updates/014.refactor_modx_allow_root.sql

```diff
@@ -0,0 +1 @@
+UPDATE modx_system_settings SET setting_value = '1' WHERE setting_name = 'udperms_allowroot';

```

Comment:
Wenn der Nutzer den Wert im MODx-Admin ändert, ist doch wieder alles kaputt, oder? Wenn ja, müsste diese Anpassung eigentlich bei jedem Update des connectors durchgeführt werden. Genauso wie alle anderen kritischen Einstellungen.

Das ExFace-Plugin ist dagegen nicht kritisch. Wenn es aus ist, passiert nix schlimmes. Dessen Einstellungen dürfen hier bleiben. 

Wichtig ist, dass wenn was in MODx kaputt gemacht wird, die Kompatibilität mit exface durch ein Repair des Connectors wiederhergestellt wird.